corehq/apps/registry/templates/registry/registry_list.html

```diff
@@ -0,0 +1,169 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load compress %}
+{% load hq_shared_tags %}
+{% load static %}
+{% load i18n %}
+
+{% block stylesheets %}{{ block.super }}
+  {% compress css %}
+  <link type="text/less"
+        rel="stylesheet"
+        media="all"
+        href="{% static 'registry/less/registry.less' %}" />
+  {% endcompress %}
+{% endblock %}
+
+{% requirejs_main 'registry/js/registry_list' %}
+
+{% block page_content %}
+  {% initial_page_data "owned_registries" owned_registries %}
+  {% initial_page_data "invited_registries" invited_registries %}
+  <div id="data-registry-list">
+    <div class="row">
+      <div class="col-sm-8 col-md-6">
+        <h3>Data Registries</h3>
+      </div>
+      <div class="col-sm-4 col-md-3">
+        <div class="pull-right" style="margin-top: 17px; margin-bottom: 8.5px">
+          <button type="button" class="btn btn-primary" data-bind="click: newRegistry">
+            <i class="fa fa-plus"></i>
+            {% trans "New Registry" %}
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-12 col-md-9">
+      <!-- ko foreach: ownedRegistries -->
+      <div class="panel panel-default show-focus">
+        <div class="panel-heading clickable">
```

Comment:
What happens when you click on the header, are these collapsible?

Reply:
It will go to the edit page - should I add a pencil icon or something?

Reply:
How about making the title text into a link, with or without a pencil? I don't think people will expect to change pages when they click a panel header, a link is a better signal that you're going to leave the page.